packages/backend/src/peripherals/database/shared/types.ts

```diff
@@ -11,8 +11,14 @@ declare module 'knex/types/tables' {
     last_updated: Date
   }
 
+  interface DiscoveryRow {
+    one_row_id: true
```

Comment:
Why `one_row_id`? 😄

Reply:
This is a trick that makes sure we store only one row in this table.